shared-components/Utils/theme.ts

```diff
@@ -1,7 +1,7 @@
 import { extendTheme } from "@chakra-ui/react";
 import { Theme } from "@nivo/core";
 import { defaultStyles } from "@visx/tooltip";
-import { percent } from "./chart";
+import { percent } from "../Statviz/utils/chart";
```

Comment:
you should not import from Statviz in shared components. Otherwise, the dependencies start to become circular.